packages/grid/_modules_/grid/components/GridScrollArea.tsx

```diff
@@ -52,6 +53,7 @@ export const GridScrollArea = React.memo(function GridScrollArea(props: ScrollAr
         api!.current.scroll({
           left: scrollPosition.current.left + offset,
           top: scrollPosition.current.top,
+          absoluteTop: scrollPosition.current.absoluteTop,
```

Comment:
So as discussed yesterday, this parameter can now go

Reply:
can you check my other comment - the short answer is that we still need to use the scroll params

Reply:
so you can pass what you need in params. `{realScroll, renderingZoneScroll}`

Reply:
Removed / updated